### PR TITLE
Testing and production grids

### PR DESCRIPTION
**Pull request by exarkun**
Fixes: ticket:144

This sets up a single node testing grid and a single node production grid.

### CODE REVIEW

File: morph/storage000-hardware.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    [ <nixpkgs/nixos/modules/installer/scan/not-detected.nix>
+    ];
+
+  boot.initrd.availableKernelModules = [ "ahci" "xhci_pci" "ehci_pci" "megaraid_sas" "usbhid" "sd_mod" ];
+  boot.initrd.kernelModules = [ ];
+  boot.kernelModules = [ "kvm-intel" ];
+  boot.extraModulePackages = [ ];
+
+  fileSystems."/" =
+    { device = "/dev/disk/by-uuid/ccabaa39-d888-467e-b8d9-75b5790a91aa";
```
Comment (jacob-ltm) [inline morph/storage000-hardware.nix]: should the UUID of the disks be hard coded?

**Comment by jacob-ltm** [comment]
Left one comment in place, but it shouldn't be a blocker.

File: morph/storage000-hardware.nix (hunk 1 above)
Comment (exarkun) [reply morph/storage000-hardware.nix]: I think it should.  This is basically equivalent to fstab.  By UUID is easiest way to unambiguously identify a disk (sdX will change if the hardware reports disks in a different order, such as if physical connections are moved around or even potentially on a firmware upgrade).  If the disk with the root partition actually dies and gets replaced we will have to change this configuration... but as things currently stand we will also have to reinstall NixOS overtop of Debian so we might just regenerate this file anyway.

**Comment by exarkun** [comment]
Thanks!

**Pull request merged**